go/obscurocommon/common.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 const (
-	L2GenesisHeight = uint64(1)
+	L2GenesisHeight = uint64(0)
```

Comment:
Why the switch to `0`?

Reply:
it has to be 1.
Because we made the height to be uint

Reply:
Fixed in https://github.com/obscuronet/obscuro-playground/pull/52